### DISCUSSION TITLE
Child categories using the same name are grouped as one in category module

### DISCUSSION THREAD

**Discussion by joomlafun** [joomla/joomla-cms#28690] [2. Cold Features]
### Steps to reproduce the issue
1° Create a joomla category module
2° In filtering option; select "inclusive" with 2 child categories having the same name.

### Expected result
The module should show 2 category groups of articles.

### Actual result
The module shows one category group with all articles of both categories in it.

### System information (as much as possible)
Joomla! 3.9.6 ; php 7.3

### Additional comments
It would be great to have the module to show the parent categories as well to differenciate the 2 child categories which have the same name.

**Comment by SharkyKZ** [comment]
Solving this would require a big change in data passed to the layout. So I don't think we can fix this in J3.

**Comment by ghost** [comment]
`New Feature`-label for J4.
